File: test/for6/main.go
package main

type Address struct {
	Number int
	Street string
	Name string
	ZipCode int
}

func main() {	
	adds := []Address{}

	for i:=0; i<10; i++ {
		temp := Address {21, "Jump st", "John Doe", 10001}
		adds = append(adds, temp)
		console.log(adds)
	}
}

File: test/for6/main.js
// package main

function Address(Number, Street, Name, ZipCode) {
    this.Number = Number;
    this.Street = Street;
    this.Name = Name;
    this.ZipCode = ZipCode;
}

function main() {
    var adds = [];
    for (var i = 0; i < 10; i++) {
        var adds = append(adds, new Address(21, "Jump st", "John Doe", 10001));
    }
}
main();

